### ISSUE TITLE
marabunta does not pass the database connexion information to odoo

### ISSUE THREAD

**Issue by vincent-hatakeyama** [camptocamp/marabunta#68]
I’ve set up this very simple marabunta.yaml:

```yaml
migration:
  versions:
    - version: setup
      addons:
        upgrade:
          - base
```

When I run marabunta by setting all the environment variables, I get:

```
|> migration: processing version setup
|> version setup: start
|> version setup: execute base pre-operations
|> version setup: execute local pre-operations
|> version setup: installation / upgrade of addons
|> version setup: odoo --workers=0 --stop-after-init --no-xmlrpc -i base
2022-03-04 08:48:05,401 11 INFO ? odoo: Odoo version 13.0 
2022-03-04 08:48:05,401 11 INFO ? odoo: addons paths: ['/usr/local/lib/python3.8/dist-packages/odoo/addons', '/var/lib/odoo/.local/share/Odoo/addons/13.0'] 
2022-03-04 08:48:05,401 11 INFO ? odoo: database: default@default:default 
2022-03-04 08:48:05,609 11 INFO ? odoo.addons.base.models.ir_actions_report: Will use the Wkhtmltopdf binary at /usr/local/bin/wkhtmltopdf 
2022-03-04 08:48:05,839 11 INFO ? odoo.service.server: Initiating shutdown 
2022-03-04 08:48:05,839 11 INFO ? odoo.service.server: Hit CTRL-C again or send a second signal to force the shutdown. 
```

I’m expecting marabunta to provide the connexion information to Odoo (it should not be default@default:default). I also haven’t found a way to provide a configuration file to odoo. Is it supposed to be a fixed configuration file and set up with migration/options/install_args? That would mean the configuration file must be present at the same place in each environment (this is something mentioned in #31 )

**Comment by vincent-hatakeyama** [comment]
I’ve tried to use `install_args` to set the path to the configuration file. It does not work, but any option set in install_args is not used. I’ve looked at the code, there is not use of this option in https://github.com/camptocamp/marabunta/blob/master/marabunta/parser.py#L152

**Comment by p-tombez** [comment]
Regarding the issue with providing a configuration file to Odoo, I just commented [here](https://github.com/camptocamp/marabunta/pull/59#issuecomment-1059005054) what's going on.
TL;DR use `install_command` to pass the arguments to Odoo.

For the issue with connection infos not sent to Odoo, can you provide exactly what command you're running ? (minus sensitive data, of course)

**Comment by vincent-hatakeyama** [comment]
Odoo and marabunta are installed (in the docker). marabunta is run by using the marabunta entry point, everything else is passed as environment variables, including the path to the migration file, db host (this is 127.0.0.1 because the docker is run with --network host), db port and db user.

**Comment by vincent-hatakeyama** [comment]
After looking at this again, it looks like Odoo accepts using environment variables to set the database user, port and host using the same variables that psql uses. Marabunta is apparently expecting those to be set, so it does not set them before running Odoo. The password can also be passed in a similar way, it is probably done in psycopg2.

It leaves only the issue of the database name. It should be provided to odoo with -d/--database by marabunta.

**Comment by vincent-hatakeyama** [comment]
There’s an alternative if the DB does not have the same name across environment. The dbname should either be in the configuration file or absent from it and passed via PGDATABASE (odoo uses this environment variable if the database name is not provided from somewhere else). It feels like a workaround but it seems marabunta expects to pass the database name in either of those way.
